site/content/about/_index.md

```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
 
 ## Team
 
-Techqueria is possible thanks to the work of lots of people, organizers, volunteers, mentors, sponsors, and advisors; and we're thankful for all of them.
+Techqueria is an all-volunteer organiztion that is made possible thanks to the work of lots of organizers, mentors, sponsors, and advisors. Thank you all!
```

Comment:
I think the "Thank you all" message isn't needed as we thank them by naming them throughout the rest of the page.